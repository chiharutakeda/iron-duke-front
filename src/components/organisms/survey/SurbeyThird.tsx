import React from 'react';
import { useRouter } from 'next/router';
const SurbeyThird = () => {
  const router = useRouter();
  return (
    <>
      <div>SurbeyThird</div>
      <button
        onClick={() => {
          router.push('SurbeyFinish');
        }}
      >
        次へ
      </button>
    </>
  );
};

export default SurbeyThird;
