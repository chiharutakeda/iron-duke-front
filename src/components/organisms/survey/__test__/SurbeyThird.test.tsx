import ReactTestRenderer from 'react-test-renderer';
import SurbeyThird from '../SurbeyThird';

it('SurbeyThird', async () => {
  const tree = await ReactTestRenderer.create(
    <SurbeyThird />
  ).toJSON();
  await expect(tree).toMatchSnapshot();
});
