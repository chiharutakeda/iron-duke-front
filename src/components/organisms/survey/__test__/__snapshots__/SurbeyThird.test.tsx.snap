// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SurbeyThird 1`] = `
Array [
  <div>
    SurbeyThird
  </div>,
  <button
    onClick={[Function]}
  >
    次へ
  </button>,
]
`;
